src/test/java/LocationsUtilityTest.java

```diff
@@ -0,0 +1,121 @@
+import com.google.common.collect.ImmutableList;
+import com.google.sps.utility.LocationsUtility;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public class LocationsUtilityTest {
+
+  private static final String PREFIX = "Location";
+
+  private static final Task TASK_WITH_NO_NOTES = new Task();
+  private static final Task TASK_WITH_NO_LOCATION = new Task().setNotes("sample notes");
+  private static final Task TASK_WITH_ONE_LOCATION =
+      new Task().setNotes("sample notes [Location: Google Kitchener] more sample notes");
+  private static final Task TASK_WITH_TWO_LOCATIONS =
+      new Task().setNotes("sample notes [Location: Google Kitchener] [Location: Google Montreal]");
+  private static final Task TASK_WITH_ONE_LOCATION_ENCLOSED_INCORRECTLY =
+      new Task().setNotes("(Location: Google Kitchener)");
+  private static final Task TASK_WITH_EMPTY_LOCATION = new Task().setNotes("[Location: ]");
+
+  private static final List<Task> NO_TASKS = ImmutableList.of();
+  private static final List<Task> TASKS_WITH_NO_NOTES = ImmutableList.of(TASK_WITH_NO_NOTES);
+  private static final List<Task> TASKS_WITH_NO_LOCATION = ImmutableList.of(TASK_WITH_NO_LOCATION);
+  private static final List<Task> TASKS_WITH_ONE_LOCATION =
+      ImmutableList.of(TASK_WITH_ONE_LOCATION);
+  private static final List<Task> TASKS_WITH_TWO_LOCATIONS =
+      ImmutableList.of(TASK_WITH_TWO_LOCATIONS);
+  private static final List<Task> TASKS_WITH_ONE_LOCATION_ENCLOSED_INCORRECTLY =
+      ImmutableList.of(TASK_WITH_ONE_LOCATION_ENCLOSED_INCORRECTLY);
+  private static final List<Task> TASKS_WITH_EMPTY_LOCATION =
+      ImmutableList.of(TASK_WITH_EMPTY_LOCATION);
+  private static final List<Task> ALL_TASKS =
+      ImmutableList.of(
+          TASK_WITH_NO_NOTES,
+          TASK_WITH_NO_LOCATION,
+          TASK_WITH_ONE_LOCATION,
+          TASK_WITH_TWO_LOCATIONS,
+          TASK_WITH_ONE_LOCATION_ENCLOSED_INCORRECTLY,
+          TASK_WITH_EMPTY_LOCATION);
+
+  @Test
+  public void getLocationNoTasks() {
+    // Obtain locations in the the task notes of no tasks.
+    Assert.assertEquals(ImmutableList.of(), LocationsUtility.getLocations(PREFIX, NO_TASKS));
+  }
+
+  @Test
+  public void getLocationNoNotes() {
+    // Obtain location in the task notes of one task with no notes defined.
+    Assert.assertEquals(
+        ImmutableList.of(), LocationsUtility.getLocations(PREFIX, TASKS_WITH_NO_NOTES));
+  }
+
+  @Test
+  public void getNoLocation() {
+    // Obtain location in the task notes of one task with no location.
+    Assert.assertEquals(
+        ImmutableList.of(), LocationsUtility.getLocations(PREFIX, TASKS_WITH_NO_LOCATION));
+  }
+
+  @Test
+  public void getOneLocation() {
+    // Obtain location in the task notes of one task with one location.
+    Assert.assertEquals(
+        ImmutableList.of("Google Kitchener"),
```

Comment:
The expected location should be a constant, so that it's defined somewhere close to where you define the test inputs.

Reply:
Agreed, the string "Google Kitchener" is also scattered throughout the file so it makes sense to be a constant. Thanks!